testing/run-scripts/serve-zork-metrics.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t zork-metrics ../../monitor
```

Comment:
We've seen some benefits to eliminating `docker build` from our scripts, in preference to uploading the images to Docker Hub. If you want to leave that for a followup, fine, but either way let's follow the naming convention of `uproxy/whatever`. I'm going to suggest `uproxy/monitor`.